test/test.js

```diff
@@ -814,3 +814,19 @@ test('without --dot option exclude folders/files with a dot', async t => {
   t.is(result.stdout, '');
   t.is(result.stderr, '');
 });
+
+test('with --quiet option does not print to stdout or stderr', async t => {
+  const fixFileQ = 'incorrect.q.mdf';
+  try {
+    fs.copyFileSync('incorrect.md', fixFileQ);
+    await execa('../markdownlint.js',
+      ['--fix', '--quiet','--config', 'test-config.json', fixFileQ],
```

Comment:
Why do we need to use --fix in this test? I think it's simpler if we don't need to copy/edit/delete the file. Though this may raise a good point that --fix should yield exitCode 0 when it's able to fix everything. I don't know if that's currently verified - it's outside the scope of this PR, tho.